tests/mbt/driver/mbt_test.go

```diff
@@ -523,6 +558,9 @@ func ComparePacketQueues(
 	consumer string,
 	timeOffset time.Time,
 ) {
+	if consumer == "consumer1" {
```

Comment:
Why is this needed here?

Reply:
Oops, a legacy left over from debugging. Thanks for catching!